woocommerce/woocommerce
# Update Google logo icon

### All Submissions:


### Changes proposed in this Pull Request:

Link to P2: pe2C5g-kK-p2#comment-160

In this PR, as per the request in the P2 above, we update the Google logo icon from the "Google shopping tag" icon to generic colorful "G" icon.

In the `google.js` file, I use `img` element with `WC_ASSET_URL` to point to the svg file, so that we don't need to have duplicated svg code in the JS file.

Additional technical changes:

- I simplified the `ProductIcon` component by using the `size` prop and removing custom CSS for svg element. The `size` prop applies to both svg and img elements.

### Screenshots

In "Recommended extensions" card:

<img width="899" alt="image" src="https://user-images.githubusercontent.com/417342/186970198-58033dbf-7791-4a85-b5bb-327c5a47befe.png">

In "Installed marketing extensions" card:

<img width="892" alt="image" src="https://user-images.githubusercontent.com/417342/186970354-9362ce6e-7c73-4a99-b95b-c18130f733ca.png">

### How to test the changes in this Pull Request:

1. Go to the Marketing page: `/wp-admin/admin.php?page=wc-admin&path=%2Fmarketing`
2. You should see the "Recommended extensions" card. Google logo icon should be as per the above screenshot.
3. Install the Google Listings and Ads plugin.
4. In the "Installed marketing extensions" card, Google logo icon should be as per the above screenshot.

### Other information:




### FOR PR REVIEWER ONLY:
